### PR TITLE
Fix integers in enumerations #1943

### PR DESCRIPTION
**Pull request by vulevukusej**

### CODE REVIEW

**Comment by Gorgious56** [comment]
#1943 just for reference

File: src/blenderbim/blenderbim/bim/prop.py [hunk 1]
```diff
@@ -147,11 +148,17 @@ class Attribute(PropertyGroup):
...
-
+    enum_data_type: StringProperty(name="Enum Data Type")
+    
     def get_value(self):
         if self.is_null:
             return None
-        return getattr(self, str(self.get_value_name()), None)
+        if self.data_type == "enum":
+            type_map = blenderbim.bim.schema.ifc.type_map
+            type_fn = {'integer':int, 'string':str, 'float':float, 'bool': bool}[type_map[self.enum_data_type]]
```
Comment (Gorgious56) [inline src/blenderbim/blenderbim/bim/prop.py:158]: This is ugly but not unprecedented https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/blenderbim/blenderbim/bim/module/patch/operator.py#L108-L113

**Comment by Gorgious56** [comment]
It makes sense to me. 

Have you been able to test it and see if there is no loss of information when exporting / importing psets with non-str enum values ?

File: src/blenderbim/blenderbim/bim/prop.py (hunk 1 above)
Comment (Gorgious56) [reply src/blenderbim/blenderbim/bim/prop.py:158]: Well I wrote it and I'm a bit ashamed ^^ so maybe we can add this lookup dictionary as a constant somewhere in the code ?

**Comment by vulevukusej** [comment]
> 

importing and exporting works fine - i've been using this commit productively for the past few days

File: src/blenderbim/blenderbim/bim/prop.py (hunk 1 above)
Comment (vulevukusej) [reply src/blenderbim/blenderbim/bim/prop.py:158]: haha - yeah it ain't pretty

**Pull request merged**